tests/Resources/ti.network.httpclient.addontest.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2019 Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Network.HTTPClient', function() {
+  this.timeout(6e4);
+
+  it.android('TLSv3 support', function(finish) {
```

Comment:
<!--
  2 failure:  tests/Resources/t..., tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.network.httpclient.addontest.js;
  Line: 15;
-->
- :no_entry_sign: tests/Resources/ti.network.httpclient.addontest.js line 15 – Expected indentation of 1 tab but found 2 spaces. (indent)
- :no_entry_sign: tests/Resources/ti.network.httpclient.addontest.js line 15 – Missing space before function parentheses. (space-before-function-paren)